lib/plugin/screenshotOnFail.js

```diff
@@ -131,4 +131,16 @@ module.exports = function (config) {
       }
     }, true);
   });
+
+  function _getUUID(test) {
+    if (test.uuid) {
+      return test.uuid;
+    }
+
+    if (test.ctx) {
+      return test.ctx.test.uuid;
+    }
+
+    return Math.floor(new Date().getTime() / 1000);
```

Comment:
Why you not used uuid4() ?

Reply:
@Vorobeyko well, it's used before like that. I just moved it to separate method. Not sure if something is broken if we use `uuid4()`?

Reply:
uuid4 is too long... better nanoid but that's an extra dependency